error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/RandomRulesTestInput.java

```diff
@@ -0,0 +1,20 @@
+package tech.picnic.errorprone.refasterrules;
+
+import com.google.common.collect.ImmutableSet;
+import java.util.Random;
+import tech.picnic.errorprone.refaster.test.RefasterRuleCollectionTestCase;
+
+final class RandomRulesTest implements RefasterRuleCollectionTestCase {
+  @Override
+  public ImmutableSet<Object> elidedTypesAndStaticImports() {
+    return ImmutableSet.of();
+  }
+
+  int testRandomNextInt() {
```

Comment:
We generally have 1 test method per Refaster rule :).

There is a `RandomRules.class` missing in the `RefasterRulesTest#RULE_COLLECTIONS` 😄. We'll see that the test will fail as it expects these tests to be combined in one method :)